src/components/schedule-page/table/schedule/schedule-parts/base-cell/base-cell-input.component.tsx

```diff
@@ -10,6 +10,7 @@ export interface BaseCellInputOptions {
   onKeyDown: (event: React.KeyboardEvent<HTMLInputElement>) => void;
 }
 
+// eslint-disable @typescript-eslint/explicit-function-return-type
```

Comment:
Zamiast tego lepiej dodać return type w 32: 
```js
      onBlur={(e): void | boolean =>
        !!e.currentTarget.value && onValueChange(e.currentTarget.value)
      }
```

Reply:
no wlasnie nie moglem tego znalesc w ktorej to linii bylo :) poza tym ten komentarz nie dzialal i musialem pushowac z --no-verify